e2e/testcases/reconciler_manager_test.go

```diff
@@ -215,7 +228,56 @@ func hasGeneration(generation int64) nomostest.Predicate {
 		return nil
 	}
 }
-
+func firstContainerTerminationMessagePathIs(terminationMessagePath string) nomostest.Predicate {
```

Comment:
How about renaming these predicates to end with -Equals/Equal? Like `containerTerminationMessagePathEquals` & `containerStdinEquals` (add a containerIndex int arg), and `tolerationsEqual`.

Reply:
Maybe `hasGeneration` -> `hasGenerationEquals` too

Reply:
Will adress this in follow up PR